app/views/projects/create.js.erb

```diff
@@ -0,0 +1,7 @@
+<% # refresh the projects list %>
```

Comment:
We don't use ERB tags inside JavaScript (okay, we do in old code, but we definitely never should). I think it's okay for this small project, but in general there is always a way to have the JavaScript work unobtrusively without needing to use ERB to inject stuff into the JS code.